apps/system/test/unit/nfc_handover_manager_test.js

```diff
@@ -94,12 +94,16 @@ suite('Nfc Handover Manager Functions', function() {
     });
 
     test('start', function() {
+      var fakePromise = new MockPromise();
+      this.sinon.stub(Service, 'request', () => fakePromise);
       this.sinon.stub(window.navigator, 'mozSetMessageHandler');
       nfcHandoverManager.start();
 
       assert.isFalse(nfcHandoverManager.incomingFileTransferInProgress);
       assert.isFalse(nfcHandoverManager.bluetoothStatusSaved);
       assert.isFalse(nfcHandoverManager.bluetoothAutoEnabled);
+      fakePromise.mFulfillToValue(MockBTAdapter);
```

Comment:
Let's check if bluetooth adapter was actually stored after the Promise was fulfilled.